modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,113 @@
+
+The CVO periodically checks the conditional updates for update risks.
+These risks are conveyed through the data served by the OpenShift Update Service, which contains information for each version about known issues that may affect a cluster updated to that version.
+Most risks are limited to clusters with specific characteristics, such as clusters with a certain size or clusters that are deployed in a particular cloud platform.
+
+The CVO continuously evaluates its cluster characteristics against the conditional risk information for each conditional update, and if it finds that the cluster matches the criteria, it stores this information in the `conditionalUpdates` field of its ClusterVersion resource.
+If the CVO finds that the cluster doesn't match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its ClusterVersion resource.
+The user interface (web console and CLI) then presents this information to the administrator.
+Each "supported but not recommended" update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
+
+You can inspect all available updates with the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --include-not-recommended
+----
+
+The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
+
+.Example output
+[source,terminal]
+----
+Cluster version is 4.10.22
+
+Upstream is unset, so the cluster will use an appropriate default.
+Channel: fast-4.11 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+
+Recommended updates:
+
+  VERSION     IMAGE
+  4.10.26     quay.io/openshift-release-dev/ocp-release@sha256:e1fa1f513068082d97d78be643c369398b0e6820afab708d26acda2262940954
+  4.10.25     quay.io/openshift-release-dev/ocp-release@sha256:ed84fb3fbe026b3bbb4a2637ddd874452ac49c6ead1e15675f257e28664879cc
+  4.10.24     quay.io/openshift-release-dev/ocp-release@sha256:aab51636460b5a9757b736a29bc92ada6e6e6282e46b06e6fd483063d590d62a
+  4.10.23     quay.io/openshift-release-dev/ocp-release@sha256:e40e49d722cb36a95fa1c03002942b967ccbd7d68de10e003f0baa69abad457b
+
+Supported but not recommended updates:
+
+  Version: 4.11.0
+  Image: quay.io/openshift-release-dev/ocp-release@sha256:300bce8246cf880e792e106607925de0a404484637627edf5f517375517d54a4
+  Recommended: False
+  Reason: RPMOSTreeTimeout
+  Message: Nodes with substantial numbers of containers and CPU contention may not reconcile machine configuration https://bugzilla.redhat.com/show_bug.cgi?id=2111817#c22
+----
+
+You can inspect the underlying availability data created by the CVO by querying the `ClusterVersion` resource:
```

Comment:
I worked with Petr to modify the lines below this comment in light of @shellyyang1989 and @evakhoni's feedback. Let me know if there are any issues with this particular approach

Reply:
I like the existing mentions of `availableUpdates` and `conditionalUpdates` in the preceding text.  And I'm [on the fence][1] about whether `oc  adm upgrade ...` examples are overly-instructive-sided distractions from the generic descriptions.  But I don't understand why we'd drop down to talking through specific `oc get clusterversion ...` incantations.  We've already given enough detail for admins who are interested in interacting with ClusterVersion to be able to find it and the relevant `status` properties.  We have customer-facing UIs like `oc adm upgrade ...` and the web-console to pretty-print that `status` information for customers.  It seems like you'd have to be really API-focused to care about the `oc get clusterversion ...` detail, and we have [existing API docs][2] for folks who are that interested.  It feels like a distraction for the folks who are not API-focused, as they skim through here wondering if we are going to drop another descriptive or conceptual nugget (which we don't do, but readers won't know that without skimming or us dropping this portion of the text).

[1]: https://github.com/openshift/openshift-docs/pull/60091#discussion_r1212354660
[2]: https://docs.openshift.com/container-platform/4.13/rest_api/config_apis/clusterversion-config-openshift-io-v1.html#status

Reply:
I meant these to serve as an examples that help going from abstract to concrete (you read a descriptive text and then you can look how what we described looks like when you peek under the hood), but I agree the doc will work even without them.